src/stores/DocumentSearchStore.ts

```diff
@@ -37,6 +37,13 @@ export class DocumentSearchStore {
             }
         )
 
+        reaction(
+            () => this.activeSearch.getCurrentHighlightIndex(),
+            (currentHighlightIndex) => {
+                this.hashStore.setHashState({ fi: currentHighlightIndex })
```

Comment:
it will be shorten anyway
```suggestion
                this.hashStore.setHashState({ findIndex: currentHighlightIndex })
```